### PR TITLE
Post-v0.0.1 Fixes - Release v0.0.2

### PR DESCRIPTION
**Pull request by xhlulu**
Making this PR to add a few things I noticed were missing. Will be shipped in v0.0.2

### CODE REVIEW

**Review by jourdain — APPROVED** [review]

**Comment by xhlulu** [comment]
@plotly/dash-core Wanted to have your thoughts on the `.npmignore` part; I noticed that dash core components only publish the JS files to `npm`.

image

But the components built with the boilerplate seems to also publish a bunch of irrelevant files when I run `npm publish`:

image

Should we just manually add all the files to npmignore, or would it be better to use some "include" scripts (similar to manifest.in)?

**Comment by jourdain** [comment]
Yes update `.npmignore` to only push the generated + src of the JS with the .map file as well.

File: .npmignore [hunk 1]
```diff
@@ -23,3 +23,10 @@ config
 .travis.yml
 CHANGELOG.md
 README.md
+
+*venv
+*__pycache__
+.circleci
+.vscode
+.github
+dist
```
Comment (jourdain) [inline .npmignore:32]: dist should be part of the npm package

Comment (xhlulu) [reply .npmignore:32]: Does `dist/` contain any `.js` file? I was under the impression only the generated wheels goes there

Comment (xhlulu) [reply .npmignore:32]: does `dist` contain any JS file? I thought only the wheels go there (might be wrong)

Comment (xhlulu) [reply .npmignore:32]: Actually, might be a better idea to use the `files` field: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

**Comment by xhlulu** [comment]
@jourdain Just realized the `utils` directory is not included (since we didn't specify it in setup packages). I'd like to merge this PR and release v0.0.2 asap so we can have the utils. Let me know if there's anything I should change first.

File: .npmignore (hunk 1 above)
Comment (jourdain) [reply .npmignore]: You don't want to ignore  the js files that should be part of the npm package. no need to add then in package.json in files.

**Pull request merged**